webui/react/src/services/decoder.ts

```diff
@@ -41,6 +41,12 @@ export const jsonToDeterminedInfo = (data: Sdk.V1GetMasterResponse): types.Deter
   };
 };
 
+export const mapV1ResourcePool = (
```

Comment:
**nit:** we should probably come to a naming convention for these types of function. We have `decode` prefixes, and XToY, etc. I like `map` it's pretty succinct. How about something like `mapSdkToResourcePool`?

Reply:
yea I agree they're all over the place

about mapSdkToResourcePool : sdk sounds short but I think these are more bindings and sdk might not be very accurate can you think of something as short that would be closer to binding?